zigbee2mqtt.json

```diff
@@ -20,9 +20,9 @@
   "pkgs": [
     "ca_root_nss",
     "curl",
-    "git",
-    "node",
-    "npm"
+    "git-lite",
+    "node14",
+    "npm-node14"
```

Comment:
IDK, maybe node 16 would better here.

Reply:
If z2m is still using 14 (especially in their Dockerfile) I think 14 is the way to go.